system/core/Config.php

```diff
@@ -322,6 +322,36 @@ public function base_url($uri = '', $protocol = NULL)
 		return $base_url.$this->_uri_string($uri);
 	}
 
+	/**
+	 * Local URL
+	 *
+	 * Returns local_url [. uri_string]
+	 *
+	 * @uses	CI_Config::_uri_string()
+	 *
+	 * @param	string|string[]	$uri	URI string or an array of segments
+	 * @param	string	$protocol
+	 * @return	string
+	 */
+	public function local_url($uri = '', $protocol = NULL)
+	{
+		$local_url = $this->slash_item('local_url');
```

Comment:
This does not work if `$config['local_url']` is not defined. I would solve this like that


```suggestion
		$local_url = $this->slash_item('local_url') ?? $this->slash_item('base_url');
```

Reply:
Done.